views/logs.ecr

```diff
@@ -9,7 +9,7 @@
     <% render "header" %>
     <main>
       <section id="livelog" class="card livelog">
-        <button id="download-logs" class="btn-primary"><a href="api/logs">Download logs</a></button>
+        <button id="download-logs" class="btn-primary"><a id="download-logs-link" href="api/logs" target="_blank">Download logs</a></button>
```

Comment:
`content-disposition: attachment` header should fix this, should be no need for a target in the link
```suggestion
        <button id="download-logs" class="btn-primary"><a id="download-logs-link" href="api/logs">Download logs</a></button>
```

Reply:
oh, ok! that makes sort of sense. alright, let it stay then!